app/javascript/submission_handling/polling.js

```diff
@@ -46,7 +51,9 @@ const pollFetch = function(arg, otherRule = null) {
         }).
         catch((error) => {
             // This would be a network error
-            reportError(error);
+            // @todo: create a logging service
```

Comment:
This really needs to get done I think. Yet another reason to try [honeybadger](https://docs.honeybadger.io/lib/javascript/index.html). Or we can roll our own.

Reply:
I agree! honeybadger would be neat! We have #207 for this right?

Reply:
Yeah I just created that issue to have a full discussion on this topic yup. Would love for a product to do this job so nicely for us. Probably won't happen though, so we'll likely have to create our own.